packages/turf-clusters-dbscan/types.ts

```diff
@@ -1,5 +1,8 @@
+/*
```

Comment:
For this package, what import was causing problems?

Reply:
Problem was cropping up with the `import RBush from "rbush"` line.

Reply:
It looks like you're need to use ts-ignore and lose the rbush types in turf-clusters-dbscan?

Following the rabbit hole of one of the links you shared @smallsaucepan I found this module `default-import`

import { defaultImport } from "default-import";
const RBush = await defaultImport(rbushDefault);

Reply:
Thanks for finding that Tim. It does do the trick though it looks like there may be some flow on effects. To use the `const RBush = await defaultImport(rbushDefault);` approach we need to change our JS target to at least es2022 to support the top level await. More so the umd package we generate for CDN usage can't have that top level await either, so it would be a change to how turf.min.js was included directly in the browser i.e. https://rollupjs.org/configuration-options/#output-format

Btw where were you seeing errors that required a ts-ignore? Previous approach seemed to work without complaining in my VSCode.

The lib/wrapper approach seems ok. We can use defaultImport, though I think that pushes us forward too quickly and I'm not sure the implications.

Reply:
No problem if can't do it.  The migration I'm doing targets es2022 so I didn't think about top-level await.  One idea, wonder if can wrap the dynamic import into a little IIFE like:

```
import { defaultImport } from "default-import";
import rbushDefault from "rbush";

const RBush = await (async () => {
  const RBush = await defaultImport(rbushDefault);
  return RBush
})()

class RBushIndex extends RBush<Feature> {
...
```

At that point you might as well keep your wrapper if it's all the same.

I guess it was a ts-expect-error that I saw, not ts-ignore.  Maybe what I saw was just the original code you'd already replaced.
![image](https://github.com/Turfjs/turf/assets/81816/a8a7c5e0-3161-47be-854f-5e5cc3ba16f9)